auto_submit/lib/git/git_utilities.dart

```diff
@@ -0,0 +1,22 @@
+// Copyright 2023 The Flutter Authors. All rights reserved.
```

Comment:
This is under `git` sub dir. Nit: change name from `git_utilities.dart` to `utilities.dart`.

Reply:
Done!